tfjs-backend-webgpu/src/webgpu_program.ts

```diff
@@ -170,11 +172,9 @@ function makeShader(
           `  return i32(globalId.x);` :
           `  let localInvocationIndex = localId.z * workGroupSizeX * workGroupSizeY +
                    localId.y * workGroupSizeX + localId.x;
-               let workGroupID = (globalId - localId)/vec3<u32>(
-                   workGroupSizeX, workGroupSizeY, workGroupSizeZ);
 
-               return i32((workGroupID.z * numWorkgroups.x * numWorkgroups.y +
-                   workGroupID.y * numWorkgroups.x + workGroupID.x) *
+               return i32((workGroupId.z * numWorkgroups.x * numWorkgroups.y +
+                   workGroupId.y * numWorkgroups.x + workGroupId.x) *
                    (workGroupSizeX * workGroupSizeY * workGroupSizeZ) +
                    localInvocationIndex);
```

Comment:
The `localInvocationIndex` can also be replaced by `localIndex`?

Reply:
Yes, that may also reduce computing cost.